src/trackers/HUNO.py

```diff
@@ -0,0 +1,304 @@
+# -*- coding: utf-8 -*-
+# import discord
+import asyncio
+from torf import Torrent
+import requests
+from difflib import SequenceMatcher
+from termcolor import cprint
+import distutils.util
+import os
+from pprint import pprint
+
+# from pprint import pprint
+
+class HUNO():
+    """
+    Edit for Tracker:
+        Edit BASE.torrent with announce and source
+        Check for duplicates
+        Set type/category IDs
+        Upload
+    """
+    def __init__(self, config):
+        self.config = config
+        self.tracker = 'HUNO'
+        self.source_flag = 'HUNO'
+        self.search_url = 'https://hawke.uno/api/torrents/filter'
+        self.upload_url = 'https://hawke.uno/api/torrents/upload'
+        self.forum_link = 'https://hawke.uno/pages/1'
+        pass
+
+    async def upload(self, meta):
+        await self.edit_torrent(meta)
+        cat_id = await self.get_cat_id(meta['category'])
+        type_id = await self.get_type_id(meta['type'])
+        resolution_id = await self.get_res_id(meta['resolution'])
+        await self.edit_desc(meta)
+        if meta['anon'] == 0 and bool(distutils.util.strtobool(self.config['TRACKERS']['HUNO'].get('anon', "False"))) == False:
+            anon = 0
+        else:
+            anon = 1
+        if bool(distutils.util.strtobool(self.config['TRACKERS']['HUNO'].get('internal', "False"))) == False:
```

Comment:
please change this internal config option to be consistent with the rest of the trackers. See below for implementation / usage: 

https://github.com/L4GSP1KE/Upload-Assistant/blob/f849e0b0e2e2ec444b42f15ec480e32122cecd76/src/trackers/AITHER.py#L83-L86 
https://github.com/L4GSP1KE/Upload-Assistant/blob/f849e0b0e2e2ec444b42f15ec480e32122cecd76/data/example-config.py#L84-L86